games-sports/speed-dreams/speed_dreams-2.2.3.recipe

```diff
@@ -0,0 +1,102 @@
+SUMMARY="Open Motorsport Simulator"
+DESCRIPTION="Speed Dreams is a 3d cross-platform, open source motorsport simulation and racing game.
+
+Speed Dreams is a fork of the open racing car simulator Torcs, aiming to implement exciting new \
+features, cars, tracks and AI opponents to make a more enjoyable game for the player, as well as \
+constantly improving visual and physics realism."
+HOMEPAGE="http://www.speed-dreams.org/"
+COPYRIGHT="2009-2018 Jean-Philippe Meuret"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://sourceforge.net/projects/speed-dreams/files/$portVersion/speed-dreams-src-base-$portVersion-r7616.tar.xz"
+CHECKSUM_SHA256="c67e48a083cd86f6c0062f0b38ed91a5d25274ea8aa656ee182f3f5074e96ff0"
+SOURCE_DIR=""
```

Comment:
Not needed:
```diff
- SOURCE_DIR=""
```

Reply:
Without that line,:
`Error: speed_dreams-2.2.3 doesn't exist in sources! Define SOURCE_DIR in recipe?`

Reply:
Really should check a build, will get back on it :)